dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/MethodInvocationNode.java

```diff
@@ -76,6 +96,28 @@ public TreePath getTreePath() {
         return treePath;
     }
 
+    /**
+     * If this MethodInvocationNode is a node for an {@link Iterator#next()} desugared from an
+     * enhanced for loop, then return the expression in the for loop, e.g., {@code iter} in {@code
+     * for(Object o: iter}. Otherwise, return null.
+     *
+     * @return the iter expression, or null if this is not a {@link Iterator#next()} from an
+     *     enhanced for loop
+     */
+    public @Nullable ExpressionTree getIterExpression() {
```

Comment:
I don't see an invocation of this method and no other read access of the field. I must be missing something.

Reply:
I failed to copy those files changes to this PR.  I've done so now, thanks for pointing this out!